region/maridia/outer/Mama Turtle Room.json

```
@@ -416,6 +448,34 @@
       "clearsObstacles": ["A"],
       "note": "Ride Mama Turtle. To avoid getting hit, shoot one of her babies to wake her up, then quickly get on her back."
     },
+    {
+      "link": [1, 4],
+      "name": "Get Back on Mama Turtle",
+      "requires": [
+        "h_canNavigateUnderwater",
+        "canUseEnemies",
+        {"or": [
+          "canTrickyDodgeEnemies",
+          {"and": [
+            "Gravity",
+            "canTrickyJump"
+          ]},
+          {"enemyDamage": {
+            "enemy": "Kame (Tatori)",
+            "type": "contact",
+            "hits": 1
+          }}
+        ]}
+      ],
+      "clearsObstacles": ["A"],
+      "note": [
+        "Requires getting back on Mama Turtle while she is at the right.",
+        "The safest way is to have a normalized fall speed with either a full height jump or falling from the ledge above.",
+        "If suitless, walking off the right ledge when she hits the left wall can be a good indicator to help land on her safely.",
+        "Shrinking Samus' hitbox after touching the turtles back will also usually help.",
+        "At the top of the ride, jump to the top right ledge."
```

```suggestion
        "Shrinking Samus' hitbox after touching the turtles back will also usually help."
```
This strat makes the most sense to me if you collect 3 and are now coming back to get 2 from below.

---

The note indicates going to 6, but if you are getting back on top of an angry turtle and are not going to 3, then being positioned for 5 or 2 makes more sense.

---

Oh, yeah. This looks like a copy paste error. Good catch